app/controllers/api/v1/medical_histories_controller.rb

```diff
@@ -5,7 +5,7 @@ class Api::V1::MedicalHistoriesController < Api::Current::MedicalHistoriesContro
   def sync_from_user
     Raven.capture_message(
       'User is trying to sync medical histories using API V1.',
-      user: current_user.id
+      extra: { user: current_user.id }
```

Comment:
There's also a way to explicitly set user context in the raven gem:

```ruby
Raven.user_context(id: current_user.id)
```

Should we do that on user login so it's auto-captured for all sentry events in the session?

Reply:
Agreed. Updated the PR.